src/Cli/problem.py

```diff
@@ -6,14 +6,21 @@
 import click
 import os
 from ..Scrapers.CFScraper import CFScraper
+from ..Scrapers.CCScraper import CCScraper
 from ..Problem.ProblemManager import ProblemManager
+from .exceptions.UnsupportedPlatform import UnsupportedPlatform
 
```

Comment:
I don't see any implementation of this class

Reply:
@akstron I created a directory named "exceptions" under "Cli" directory, the same directory where "problem.py" exists. I have implemented this class in the UnsupportedPlatform.py file under exceptions.

Reply:
I apologize for the error from my side @akstron . Looks like the "exceptions" directory containing the class implementation was left out of the commit. I have fixed that in the latest commit.